fix  : Woocommerce button does not inherit global options (#4954)
brainstormforce/astra: changelog.txt

- Fix: PHP error 'Uncaught TypeError: ltrim()' for getting taxonomy term link.
- Fix: Single post title transformed to capitalize after 4.0 update.
- Fix: WooCommerce - Console throwing an event listener error when a product has only 1 item in stock.

> - Fix: Customizer - Typography and Button Preset options selected preset not highlighting and reset button not functioning.
> - Fix: WooCommerce - Button's does not inherit global button styling.

Reviewer: ```suggestion
- Fix: WooCommerce - Buttons does not inherit global button styling from the theme customizer.
```
